TheThingsNetwork/lorawan-stack
# Support setting multiple frequency plans for gateways from the Console

#### Summary



Closes #6580 

#### Changes



- Add gateway frequency plan select field component
- Update entry and key value map components to allow updating options
- Refactor fetch select
- Update validation schema
- Display the various frequency plan ids in gateway overview and in the general settings
- Fix bytes component (error in the console)

https://github.com/TheThingsNetwork/lorawan-stack/assets/56658938/a8b13f42-5bee-4460-9a9a-a6b417d910ef


#### Testing



1. Go to register a gateway.
2. Add a couple of frequency plans and notice that the already selected plans are not in the options list.
3. If no frequency plans id is the form submit should not be possible.
4. Registering the gateway should be successful.
5. In the gateway overview see the selected frequency plans.
6. In the general settings change some of the frequency plans, refresh the page and see the newly selected frequency plans.

#### Checklist
